### ISSUE TITLE
rename repository into project

### ISSUE THREAD

**Issue by DavidBadura** [DavidBadura/git-webhooks#9]

**Comment by tolry** [comment]
Be carefull, this is a BC break. Since there have been no releases until now, this might be not that big of a deal, but having a version number might be a good idea from now on.

**Comment by tolry** [comment]
In addition: on MergeRequest events this might be not that easy. There is always just one project, but there might be two repositories involved. So maybe splitting project and repository information might be the way to go?

**Comment by DavidBadura** [comment]
> on MergeRequest events this might be not that easy. There is always just one project, but there might be two repositories involved.

No, thats not right. You can only create a MergeRequest when you create a Branch in the same Project or when you fork the Project. In this case you have a new Project. So splitting project and repository is not neccessery and i think it is simpler to keep one class "Project".

**Comment by tolry** [comment]
Ok, so the current state is: if I create a merge request from tolry/simpspector#mybranch to simpspector/simpspector#master this will create a new project?

**Comment by tolry** [comment]
Just realized my problem above is probably very centered on SimpSpector, not on git-webhooks :-D

**Comment by DavidBadura** [comment]
git-webhooks is centered around github, gitlab and bitbucket. And yes, if you create a merge request from `tolry/simpspector#mybranch` to `simpspector/simpspector#master` then you have two github projects called `tolry/simpspector` and `simpspector/simpspector`.
